%%%  \textbf{\textit{What processes control the generation of biodiversity?}}
%%%  Elucidating answers to this question, in its broadest sense, is the primary
%%%  motivation of my lab group.
%%%  % To address this question, I use patterns of genetic variation within and among
%%%  % natural populations to identify independent evolutionary lineages (i.e.,
%%%  % species) and infer their demographic histories and relationships, testing for
%%%  % patterns predicted by current ecological factors and historical events.
%%%  We use a broad suite of methods in this endeavor, including
Shared ancestry is a fundamental property of life.
By providing a framework of accounting for the inherent
% nonindependence
% dependence
covariation
in biological data created by shared evolutionary history,
phylogenetics is rapidly becoming the statistical foundation of comparative
biology.
While research interests in my lab
are very diverse,
% span biogeography, speciation, and molecular and genome evolution,
\textbf{\textit{all of our work is unified by our common goal of using
        phylogenetics to infer and explicitly account for shared ancestry in
        our approaches to better understand
        processes of evolution and biological diversification}}.
        % evolution and biodiversity}}.
My lab uses a broad suite of methods in this endeavor, including
% collection-based fieldwork,
% fieldwork,
% collecting genome-wide DNA sequence data via next-generation sequencing (NGS)
% technologies,
% next-generation sequencing (NGS),
% high-throughput DNA sequencing,
% collecting genomic data,
field and lab work,
developing statistical procedures for inferring evolutionary history,
% from genomic datasets,
implementing such methods in software,
and ultimately applying these computational tools to genomic data to test
hypotheses about processes of evolution and diversification.
% Given the integrative nature of my research program, I seek to recruit students
% with diverse backgrounds and interests.
% The questions I am passionate about exploring are open-ended and require
% critical thinking and creativity.
% As a result, it is imperative that I involve students and collaborators with
% diverse backgrounds and perspectives to bring together unique insights.
The questions my lab explores are integrative, open-ended, and require critical
thinking and creativity.
As a result, it is imperative that I involve students and collaborators with
diverse backgrounds and perspectives to bring together unique insights.

\section*{Previous Research}
\subsubsection*{Climate-driven diversification}
One question I am passionate about is whether Quaternary climatic oscillations
promoted diversification by fragmenting the distributions of species.
An ideal system for addressing this question is the dynamic landscape of
Southeast Asia, where many groups of islands coalesced during lower sea levels
of glacial periods and were fragmented during interglacial rises in sea level.
It has often been assumed that this repeated fragmentation functioned as a
``species pump,'' driving vicariant speciation across these islands.
If correct, this predicts that population divergences across islands fragmented
by rising sea levels were temporally clustered around interglacial sea
incursions.
% To test this prediction, I compared the divergence times of inter-island
% populations across 22 tetrapod taxa.
To test this prediction, we collected DNA sequences to infer and compare the
divergence times among 22 pairs of populations on islands that were connected
during glacial periods.
Using an existing approximate-Bayesian computation (ABC) method, we found very
strong support
% (posterior probability of 0.982)
for a single, recent, simultaneous divergence event shared by all 22 taxa.

Suspicious of such strong support given the richness and stochasticity of the
model underlying the ABC method, I used computer simulations to confirm that
the method often supports highly clustered models of divergence even when taxa
diverged randomly over millions of years\footnote{\label{Oaks12}\shortfullcite{Oaks2012}}.
I formulated the evolutionary model underlying the simulation-based ABC
procedure and, using first principles of probability, identified theoretical
reasons the method would tend to spuriously infer simultaneous
divergences\cref{Oaks12}.
Guided by these insights, I developed a new model that avoids these theoretical
pitfalls,
implemented it in the software package
\href{https://github.com/joaks1/dpp-msbayes}{dpp-msbayes},
and developed a much more efficient 
\href{https://joaks1.github.io/PyMsBayes/}{multi-processing interface}
for this type of comparative phylogeographic inference.
As predicted by theory, the new ABC method was more
accurate\footnote{\label{Oaks14dpp}\shortfullcite{Oaks2014dpp}}.
However, the new approach also revealed that after distilling the sequence data
into insufficient summary statistics, ABC approaches were left with little
information to provide biological insights,
which prevented us from evaluating the tenability of the climate-driven
species-pump model of diversification.
% project repo: https://github.com/joaks1/msbayes-experiments

Motivated by my unsatisfied curiosity, I
(1) developed a full-likelihood Bayesian method in the software package
\href{http://phyletica.org/ecoevolity/}{ecoevolity}
that can leverage all of the information from genomic data to infer
co-divergences\footnote{\label{Oaks18ecoevolity}\shortfullcite{Oaks2018ecoevolity}},
and
(2) collected thousands of loci from across the genomes of geckos sampled from
16 inter-island pairs of populations.
The results strongly support that each pair of gecko populations diverged
independently, refuting the predictions of the climate-driven species-pump
model\footnote{\label{Oaks18paic}\shortfullcite{Oaks2018paic}};
a result that makes sense in light of the life history and habitat
preferences of these lizards.
% project repo: https://github.com/phyletica/gekgo

The entire arc of this line of inquiry was funded by NSF
(DEB
\href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1011423}{1011423},
\href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1308885}{1308885},
and
\href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1656004}{1656004})
and is a good example of my lab's willingness to derive and implement new
models and computational approaches to interrogate questions that motivate us.
I also documented our progress in all aspects of this work in open-science
project repositories that can be found at
\href{https://github.com/joaks1/msbayes-experiments}{github.com/joaks1/msbayes-experiments},
\href{https://github.com/phyletica/ecoevolity-experiments}{github.com/phyletica/ecoevolity-experiments},
and
\href{https://github.com/phyletica/gekgo}{github.com/phyletica/gekgo}.


%   Current work
    %   Generalizing Bayesian phylogenetics (NSF-funded)
        %   We are making assumption in phylogenetics
        %   Working to relax this assumption and explore full space of phylogenetic
        %   trees
        %   This will develop a statistical foundation for testing predictions of
        %   many interesting biological processes
        %   Genome evolution, epidemiology, biogeography
        %   Applying to gekkonid genomic data to test hypotheses about biogeography

    %   Generalizing the birth-death process

    %   Coevolution effect
        %   Resubmitting to NSF EEID this Fall

    %   Continuing work on shared divergences

    %   Conservation genetics

\section*{Current and Future Research}
% Generally, my lab is developing and implementing novel evolutionary genetic
% methods,
% collecting genomic datasets from 
%biodiverse regions of the planet,
% diverse empirical systems,
% and applying the former to the latter to
% learn about processes of evolution and diversification.
%across regions characterized by high levels of species richness and endemism.
% Below I detail one such method, and three empirical systems that
% have inspired its development.

\subsubsection*{Generalizing Bayesian phylogenetics}
In phylogenetics, we have long made a limiting assumption about the processes
that cause evolutionary lineages (whether they are genes or populations) to
diversify: \textit{we assume these processes affect only one lineage and cause
    it to leave only two descendants}.
Violations of this assumption are likely common and span many areas of biology,
including biogeography, gene-family evolution, epidemiology, and symbiont
coevolution.
% In biogeography, if a landscape is fragmented into more than two fragments,
% this could cause more than two lineages descending from a node, and also
% the timing of the divergence could be shared across multiple co-distributed
% taxa.
% In genome evolution, if a region of a chromosome is duplicated that contains
% multiple members of a gene family, there will be multiple, simultaneous
% divergences in the evolutionary history of this gene family.
Currently, we lack a statistical framework for testing the predictions of the
diverse and interesting processes that violate this assumption.
My lab is currently funded by NSF
(\href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1656004&HistoricalAwards=false}{DEB 1656004})
to create such a framework by generalizing the space of phylogenies that is
explored during Bayesian phylogenetic inference.
By exploring the full space of phylogenies, we can approximate the probability
of patterns predicted by processes that cause nonindependent divergences 
across the tree of life.

We have implemented this approach in 
\href{http://phyletica.org/ecoevolity/}{ecoevolity}
and are currently testing it using simulations.
To use this method to test several long-standing biogeographical
hypotheses across Southeast Asia,
we are currently collecting genomic data from several genera of geckos that are
co-distributed across much of this region.
To make our approach more generally applicable to questions about biogeography,
genome evolution, epidemiology, and coevolution,
we are also porting our distributions and algorithms to the flexible
statistical phylogenetic software package
\href{https://revbayes.github.io/}{RevBayes}.

\subsubsection*{Generalizing the birth-death process}
We have developed the theory and computational machinery to explore
the full space of phylogenetic trees, but due to the lack
of biologically inspired mechanistic models whose distribution covers
this space, we are currently resigned to use phenomenological models
to represent our \emph{a priori} assumptions.
To remediate this limitation, my lab is working in collaboration with Sebastian
H{\"o}hna on generalizing the birth-death process to allow nonindependence of
divergence times across a phylogeny.
To fund this work, we are preparing a proposal for the Systematics and
Biodiversity Science Cluster at the NSF DEB. 

\subsubsection*{An evolutionary approach to emerging infectious diseases}
Globally, the association between habitat fragmentation and the emergence of
zoonotic infectious diseases in human populations has motivated research into
the ecological processes responsible for this pattern.
Sarah Zohdy, Tonia Schwartz, and I have proposed an evolutionary mechanism for
this pattern that can complement ecological
perspectives\footnote{\label{Zohdy19}\shortfullcite{Zohdy2019}}.
As gene flow decreases between habitat fragments, the populations inhabiting
them will begin to diverge due to mutations and genetic drift.
In addition, the lack of gene flow allows mammalian hosts, obligate
ectoparasites, and pathogens within each fragment to coevolve largely
independently.
The neutral divergence among fragments, combined with the decoupling of
coevolutionary selective pressures, can cause the overall genetic
diversity of pathogens across the landscape to rapidly increase,
which raises the chances of pathogen variants with zoonotic potential.
We are working on testing this model using mouse lemurs, lice, and viruses in
Madagascar.
We received favorable reviews of our proposal to the NSF Ecology and Evolution
of Infectious Diseases (EEID) program in Fall 2017, and we are currently
preparing to re-submit this proposal in November.

% \subsubsection*{Diversification in West-Central African rainforests}
% I am working with Adam Leach\'{e} and Matthew Fujita to apply the novel
% statistical methods to comparative genomic datasets to elucidate the historical
% processes of diversification and community assembly across West-Central African
% rainforests.
% We are particularly interested in determining how Pliocene and Pleistocene
% aridification cycles and associated rainforest fragmentation influenced
% diversification across the Afro-tropics.


% \section*{Future Research}

\subsubsection*{Improving phylogenetics with genomic data}
By analytically integrating over the evolutionary history of individual loci,
my lab's software
\href{http://phyletica.org/ecoevolity/}{ecoevolity}
can infer phylogenies from millions of sites from across the genomes sampled
from natural populations.
However, as the number of taxa (and thus the size of the phylogeny) increases,
the method becomes computationally expensive.
There are a number of ways that we plan to extend this method to accommodate
larger phylogenies, including
(1) analytically integrating the population sizes along the tree to
avoid having to sample them with computationally expensive algorithms,
(2) analytically integrating variation in the rate of substitution
across the tree to relax the assumption of a strict clock while
minimizing computational burden,
and
(3) limit the size of the state space that needs to be considered when
calculating the likelihood toward the root of the tree.
These advances are imperative for inferring large phylogenies from genomic data
while explicitly accounting for the variation in evolutionary history across
chromosomes.
We will continue to seek funds from NSF to support this work.


\subsubsection*{Bringing phylogenetics into the inference of population structure}
Gideon Bradburd and colleagues recently introduced a method for assigning
individuals to populations using genomic data, while simultaneously accounting
for both continuous and discrete genetic variation across a
landscape\footnote{\label{Bradburd18}\shortfullcite{Bradburd2018}}.
This was an important step forward from methods that ignored continuous genetic
differentiation.
However, their model assumes that the allele frequencies in each population
has drifted away from an ancestral population independently of one another
(i.e., ignoring shared ancestry among the populations).
Gideon and I are working on extending the method implemented in conStruct
to simultaneously infer
(1) the population assignment of each individual's genome (allowing their
genomes to be admixed), and
(2) the phylogenetic relationships of the populations.
We are planning to submit a proposal to the NSF DEB Systematics and
Biodiversity Science Cluster in Spring 2020 to fund this work.


    %   Continued refinement of testing for shared divergences
        %   More nonparametric models
        %   Analytical integration of population sizes
        %   Modeling linked loci
        %   Gene flow
        %   Current approach ignores gene flow and assumes SNPs
        %   Working with Yujin to get at this

    %   Functional phylogeography
        %   Phylogeography with whole genomes creates opportunities to infer
        %   spatial patterns of selection
