\documentclass[10pt]{article}
%\usepackage{anysize}
%\papersize{11in}{8.5in}
%\marginsize{1in}{1in}{.5in}{.5in}
\textwidth = 6.5 in
\textheight = 9 in
\oddsidemargin = 0.0 in
\evensidemargin = 0.0 in
\topmargin = -0.5 in
\headheight = 0.35 in
\headsep = 0.15 in
\topskip = 0 in
\footskip = 0.5 in
\pagenumbering{arabic}
\usepackage{setspace}
\usepackage[usenames]{color}
\usepackage[fleqn]{amsmath}
\usepackage{graphicx}
\usepackage{url}
\usepackage{verbatim}
\usepackage{indentfirst}
\usepackage{booktabs}
\usepackage{multirow}
\usepackage[table]{xcolor}
\usepackage{ragged2e}
\usepackage{xspace}
\usepackage{parskip}
\usepackage{tabulary}
\usepackage[normalem]{ulem}
\usepackage{hyperref}
\hypersetup{pdfborder={0 0 0}, colorlinks=true, urlcolor=blue, linkcolor=black}
\usepackage[format=plain, labelsep=period, justification=raggedright, singlelinecheck=true, skip=2pt, font={footnotesize,sf}, labelfont=bf]{caption}
\usepackage{titlesec}
\usepackage{lastpage}
\usepackage{fancyhdr}
\usepackage{ifthen}
\usepackage{wrapfig}

%\usepackage[round]{natbib}
%\bibliographystyle{evolution}
\usepackage[style=nature]{biblatex}
\bibliography{references}

%% Format headers and footers %%%%%%%%%%%%%%%%%%%%
\pagestyle{fancy}
%\lhead{\ifthenelse{\value{page}=1}{}{\sffamily\footnotesize Jamie Oaks}}
\lhead{\sffamily \emph{\docTitle} \\ Jamie R. Oaks}
%\chead{\ifthenelse{\value{page}=1}{{\scshape \docTitle} \\ Jamie Richard Oaks}{\sffamily\footnotesize \docTitle}}
%\rhead{\ifthenelse{\value{page}=1}{}{\sffamily\footnotesize \today}}
\rhead{\sffamily \today}
\cfoot{\sffamily\footnotesize Page \thepage\ of \pageref{LastPage}}
\renewcommand{\headrulewidth}{0.4pt}
\renewcommand{\footrulewidth}{0pt}

%% Format section titles %%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand\refname{Peer-reviewed Publications}

\titleformat{\section}[hang]
    {\large\sffamily\bfseries}
    {\S\ \thesection.}{.5em}{}[]
\titlespacing{\section}
    {0mm}{1.0ex plus .1ex minus .1ex}{-0.5ex}

\titleformat{\subsection}[hang]
    {\large\sffamily\itshape}
    {\S\ \thesection.}{.5em}{}[]
\titlespacing{\subsection}
    {0mm}{1.0ex plus .1ex minus .1ex}{-0.5ex}

\titleformat{\subsubsection}[runin]
    {\sffamily\bfseries}
    {\S\ \thesection.}{.5em}{}[.---]
\titlespacing{\subsubsection}
    {\parindent}{0pt}{0pt}
%    {\parindent}{1.0ex plus .1ex minus .1ex}{0pt}

%% Format list environments %%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\labelenumii}{\arabic{enumi}.\arabic{enumii}}
\renewcommand{\labelitemi}{$\circ$}

\newenvironment{myEnumerate}{
  \begin{enumerate}
    \setlength{\itemsep}{0.25em}
    \setlength{\parskip}{0pt}
    \setlength{\parsep}{0.5em}}
  {\end{enumerate}}

\newenvironment{myItemize}{
  \begin{itemize}
    \setlength{\leftskip}{-4mm}
    \setlength{\itemsep}{0.25em}
    \setlength{\parskip}{0pt}
    \setlength{\parsep}{0.5em}}
  {\end{itemize}}

%% Basic formatting and spacing %%%%%%%%%%%%%%%%%%%%%
\setlength{\parindent}{0em}
\setlength{\parskip}{0.5em}

%% My functions %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\ignore}[1]{}
\newcommand{\addTail}[1]{\textit{#1}.---}
\newcommand{\super}[1]{\ensuremath{^{\textrm{#1}}}}
\newcommand{\sub}[1]{\ensuremath{_{\textrm{#1}}}}
\newcommand{\dC}{\ensuremath{^\circ{\textrm{C}}}}
\newcommand{\tableSubItem}{\addtolength{\leftskip}{1em} \labelitemi \xspace}
\newcommand{\myHangIndent}{\hangindent=5mm}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\docTitle}{Research Statement\xspace}
\begin{document}
\raggedright
\singlespacing

\textbf{\textit{What processes shape genetic variation both temporally and
        spatially, partition evolutionary lineages, and generate and maintain
        biodiversity?}} \\
\textbf{\textit{What processes control the generation and assembly of
        biodiversity?}} \\
Elucidating answers to this question is the primary motivation of my research
program.
To address this question, I use patterns of genetic variation within and among
species to identify independent evolutionary lineages (i.e., species) and infer
their demographic histories and relationships, testing for patterns predicted
by current ecological factors and historical events.
I use a broad suite of methods in this endeavor, including
fieldwork,
collecting genome-wide DNA sequence data via next-generation sequencing (NGS)
technologies,
developing model-based statistical procedures for inferring evolutionary
history,
developing computational and bioinformatics tools to implement these
statistical methods and ease data processing and management,
and ultimately applying novel methods to genomic data to determine the
evolutionary model that best explain the data.

\section*{Previous Research}
%%%%%%%%%%%%%%%%%%%%%
% \subsection*{Diversification of Crocodiles}
% Traditionally, crocodiles (\emph{Crocodylus}) have been stereotyped as an
% ancient group of ``living fossils'' that originated in Africa prior to the
% fragmentation of Pangea; their diversity and circumtropical distribution was
% attributed to vicariance via continental drift.
% However, early molecular data and subsequent reassessment of paleontological
% data suggested the genus could be younger than previously believed.
% Recently, I collected a large multi-locus dataset of all extant crocodylian
% species and used novel statistical phylogenetic methods to infer the temporal
% and biogeographical origin of \emph{Crocodylus}.
% My results overturned traditional views of crocodiles as ``living-fossils''
% from Africa, revealing a recent and dynamic evolutionary history
% \footfullcite{Oaks2011}.
% My results strongly support that all extant crocodiles shared a common ancestor
% from the tropics of the Indo-Pacific, approximately 14--8 million years ago
% (mya), rejecting an ancient vicariant explanation of their biogeography in
% favor of a recent, dispersal-mediated model involving multiple transoceanic
% dispersals.
% It was not until after the mid-Miocene climatic optimum, when the global
% climate was cooling and fellow crocodylian lineages were suffering massive
% extinctions, that \emph{Crocodylus} radiated and dispersed around the globe.
% This finding suggests it was not only global cooling driving the extinction of
% other crocodylians, as previously believed, but also most likely competition
% with the rapidly radiating \emph{Crocodylus} lineage.
% %My results also revealed more species diversity within the \emph{Crocodylus}
% %than currently recognized.

% This study also introduced important methodological innovations.
% To accommodate the variation of substitution rates across DNA positions
% within and among genes, I used a novel approach of estimating the optimal model
% for partitioning the alignment into rate categories.
% The resulting model performed far better than traditional methods that
% partition the alignment based on \emph{a priori} expectations of rate variation
% \footfullcite{OaksInPrep}.
% This has important implications, because improved modeling of among-site rate
% variation will mitigate the underestimation of long branches and concomitant
% systematic error in phylogenetic estimates (e.g., long-branch attraction).
% Furthermore, the study was among the first to estimate a time-calibrated
% species tree using a multi-species coalescent model, and was the first to use a
% posterior sample of species trees to estimate ancestral-state reconstructions.
% % The species tree represents the evolutionary history of populations, rather
% % than the history of individual gene copies.
% Using a direct estimate of the species phylogeny, rather than the history of
% individual gene copies, is ideal when inferring the evolutionary patterns of
% species-level characters, like ancestral ranges.

\subsection*{Climate-Driven Diversification}
One question I am passionate about is whether Quaternary climatic
oscillations promoted diversification by fragmenting the distributions of
species.
% One focus of my current research seeks to determine whether Quaternary climatic
% oscillations promoted diversification by fragmenting the distributions of
% species.
An ideal model system for addressing this quesiton is the complex system of
island archipelagos in Southeast Asia.
The Southeast Asian mainland and complex system of 26,000+ islands experienced
dramatic cyclical shifts in the extent of the terrestrial landscape during
Quaternary glacial cycles (Figure \ref{map}).
Many groups of islands in this region coalesced into aggregate islands during
glacial periods when sea levels were up to 120m below current level; these
islands where repeatedly fragmented during interglacial rises in sea level.

To test whether the repeated formation and fragmentation of island complexes
promoted diversification, collaborators and I have been studying the
evolutionary history of two genera of geckos (\emph{Cyrtodactylus} and
\emph{Gekko}) that are co-distributed across most of the islands of the
Philippines\footfullcite{Siler2010}\super{,}\footfullcite{Siler2012}.
We found that island complexes did explain a significant proportion of the
genetic diversity, but populations from PAICs were not monophyletic, and there
was no obvious pattern of diversification associated with the Pleistocene.
We revealed complex histories for both genera that contradict many of the
traditional areas of endemism predicted by Pleistocene island complexes.
Interestingly, for \emph{Gekko}, we found evidence that the genus colonized the
Philippines by ``rafting'' on the Palawan Microcontinental Islands that rifted
away from Mainland Asia and drifted toward the rest of the Philippine
Archipelago between 30--10 mya.

\begin{wrapfigure}{r}{0.54\textwidth}
  \vspace{-1.5em}
  \begin{center}
    \includegraphics[width=0.53\textwidth]{sunda_shelf_small.pdf}
  \end{center}
  \vspace{-0.2em}
  \caption{Map of Southeast Asia with land extent (depicted in grey) during
  glacial lowstands estimated using 120m bathymetry (data from
  \href{http://ngdc.noaa.gov/mgg/global/global.html}{ETOPO1}).}
  \label{map}
  \vspace{-1.1em}
\end{wrapfigure}

In order to more directly test whether Quaternary glacial cycles promoted
diversification, we used a broadly comparative approach\footfullcite{Oaks2012}.
we accumulated genetic data from 22 distantly related
taxa, representing five orders of terrestrial vertebrates.
For each taxon, we sampled genetic data from a pair of populations
currently on two separate islands previously connected during glacial
periods.
If repeated bouts of island connectivity and isolation promoted
diversification, the temporal distribution of divergences across the 22
inter-island pairs of populations should be temporally clustered and correspond
with interglacial rises in sea level.
To test this prediction,
we employed a popular Bayesian model-choice method, msBayes, to estimate the
probabilities of models in which multiple sets of taxa diverge at the same time.
We found very strong support
% (posterior probability of 0.982)
for a single, recent, simultaneous divergence event shared by all 22 taxa.

Rather than interpret this support for simultaneous divergence across the 22
taxa as strong evidence of climate-driven diversification, we performed
extensive simulation-based analyses to assess the ability of the msBayes method
to detect variation in divergence times.
The results of the simulations revealed that the method will often strongly
support highly clustered models of divergence even when pairs of populations
diverged randomly over millions of generations.
Thus, our results were likely spurious.
This finding is important, because msBayes is a popular method, and previous
inferences of clustered divergence times across co-distributed taxa have often
been interpreted as evidence for a shared historical event.
Our results demonstrate that the method can be biased toward such results, and
the interpretation of a shared event is not warranted without simulation-based
validation of the method's performance.

% Needed better method---did it
I formulated the evolutionary model underlying the msBayes procedure and, using
fundamental principles of Bayesian model choice, identified theoretical reasons
the method might tend to spuriously infer biogeographically interesting
scenarios of
divergence\footfullcite{Oaks2012}\super{,}\footfullcite{Oaks2014reply}.
Guided by these insights, I developed a new model that uses more flexible prior
probability distributions on many of the models' parameters, and a flexible,
nonparametric, Dirichlet-process prior across all possible divergence models.
I implemented the new model in the software package dpp-msbayes, and developed
a much more efficient multi-processing interface for both dpp-msbayes and
msBayes, PyMsBayes.
Using analyses of simulated and empirical data, I found, as predicted by theory,
the new method was much more robust\footfullcite{Oaks2014dpp}.

Under the new method, there is a lot more ambiguity. A lot of uncertainty
about the number of shared divergence events across our 22 taxa. Simply not
enough power given our data.


\section*{Current Research}
%%%%%%%%%%%%%%%%%%%%%
Generally, I am developing and implementing novel statistical methods,
collecting comparative genomic datasets from biodiverse regions of the planet,
and applying the former to the latter to enable unprecedented power to
understand the historical processes of diversification across a regions
characterized by high levels of species richness and endemism.

% Full likelihood method
\subsection*{Implementing a full-likelihood Bayesian comparative
    phylogeographic model}
While the method implemented in dpp-msbayes is more powerful and robust than
its predecessor, both methods are based on approximate likelihoods and discard
information by reducing the genetic data to a set of insufficient summary
statistics.
Thus, dpp-msbayes still struggles to discriminate among models of divergence at
the recent evolutionary timescales \footfullcite{Oaks2014dpp}.
Given that such statistical tool is integral to the evolutionary questions
I seek to answer in my empirical research,
I am currently developing a method that employs a similar hierarchical model in
a full-likelihood Bayesian framework.
I am utilizing recent developments that analytically integrate over gene trees
for biallelic data\footfullcite{Bryant2011}.
This will remove the gene trees from the numerical approximation machinery and
allow efficient sampling of divergence models and population divergence times
using full likelihoods and Markov chain Monte Carlo.
By utilizing all of the information in genome-wide sequence variation, this
method promises to be much more powerful than its approximate Bayesian
counterparts.
The new method will also be applicable at deeper evolutionary timescales.
This new tool will allow evolutionary biologists to better leverage comparative
genetic data to assess the affects of regional and global biogeographical
processes on biodiversity.

% Apply new method to genomic data from Philippine geckonids
\subsection*{Next-generation sequencing of geckos}
% (2) collecting genome-scale data from dense sampling of \emph{Cyrtodactylus}
% and \emph{Gekko} from across the Philippine Archipelago using next-generation
% sequencing technology.
In addition to developing a full-likelihood Bayesian method, I have also
collected genome-wide sequence data from nearly 300 \emph{Cyrtodactylus} and
\emph{Gekko} individuals from across the Philippine Archipelago using
NGS technology.
This dataset will be ideal for the new method, and will hopefully allow
us to better assess the affect of sea-level fluctuations on the
diversification of oceanic islands of Southeast Asia.

As a compliment to this work on oceanic islands, I am also working with
collaborators to better understand diversification in mainland Southeast Asia.
Dr. Lee Grismer, Professor of Biology at La Sierra University, and
international colleagues
Dr. Norhayati Ahmad, Professor of Science and Technology at the Universiti
Kebangsaan Malaysia (UKM), and
Dr. Shahrul Anuar Mohd Sah, Professor of the Biological Sciences at the
Universiti Sains Malaysia (USM).
Dr.\ Cameron Siler.
Dr.\ Anchalee Aowphol, Department of Zoology, Kasetsart University.
We have collected an impressive comparative dataset of reptiles and amphibians
from across much of the Sunda Shelf.
Along with further fieldwork, we are currently collecting sequence data
from many of these taxa.
Isthmus of Kras
Whether important current and paleo rivers are important biogeoraphic breaks.
% Apply new method to Central Asia (rock/sand species diversification)---nat
% geo grant

\subsection*{Diversification of the highly endemic fauna of the Gobi Desert}
The Gobi is the largest desert in Asia and is a mosaic of rocky mountain ranges
surrounded by basins of desert sand dunes.
I am interested in better understanding the origins of the unique and highly
endemic biodiversity of this region.
An international team of collaborators
(Jesse Grismer and Dr.\ Rafe Brown of the University of Kansas,
Dr.\ Natalia Ananjeva of the Russian Academy of Sciences,
Dr.\ Xianguang Guo of the Chinese Institute of Biology, and
Dr.\ Nyamsuren Batsaikhan of the National University of Mongolia)
and I were awarded a grant from the National Geographic Society to better
document and understand the history of the biodiversity of this region.
We are interested in the evolutionary origins of the unique taxa that
inhabit this region.
Our goal is to collect genomic data from a set of focal sand- and rock-adapted
taxa and use the comparative phylogeographical methods I am developing to
(1) infer their evolutionary origins,
(2) test for patterns of shared evolutionary history, and
(3) identify key processes that lead to the unique fauna of this region.

% Unfunded collaborator on Adam and Matt's pending NSF proposal to apply method
% Africa
\subsection*{Application of novel methods to understand diversification
in West-Central African rainforests}
I am working with Drs.\ Adam Leach\'{e} and Matthew Fujita to apply the novel
statistical methods described above to comparative phylogeographic NGS datasets
to elucidate the historical processes of diversification and community assembly
across West-Central African rainforests.
We are particularly interested in determining how Pliocene and Pleistocene
aridification cycles and associated rainforest fragmentation influenced
diversification across the Afro-tropics.
The combination of NGS data with the improved methods of inference will enable
unprecedented power to understand the historical processes of diversification
across a region characterized by high levels of species richness and endemism.
I am a collaborator on an NSF proposal to extend this work that was recently
submitted by Drs. Leach\'{e} and Fujita.

% Apply new method to sunda shelf (Malaysia/Thailand)

\section*{Future Research}
%%%%%%%%%%%%%%%

% Extend msBayes-like model to fully phylo
The evolutionary genetics software package, BEAST
(\url{http://code.google.com/p/beast-mcmc/}), implements a Bayesian
multi-species coalescent model that jointly estimates multiple gene trees and
the shared species tree in which they evolve.
In this model, the divergence times (node heights of the rooted, ultrametric
species tree) are among the parameters that are jointly estimated.
As a developer for BEAST, I am implementing a novel hierarchical Bayesian model
for co-estimating the number of divergence events across a species tree, the
assignment of internal nodes to these events, and the timing of each event from
genomic data, while integrating over uncertainty in phylogeny, gene trees, and
demographic and mutational parameters.
The resulting method will provide a fully Bayesian statistical framework
for determining whether the temporal pattern of divergences across taxa are
consistent with predictions of historical processes.

This model will have several advantages over current methods.
It will take advantage of the full information in the sequence data by
estimating the joint posterior distribution of the model directly from the
sequence data, avoiding the loss of information suffered by ABC when distilling
the data into summary statistics.
The new method will estimate the relative mutation rates and population sizes
across the species tree using rigorous relaxed-clock and demographic models
already implemented in BEAST.
% This will provide more accurate estimates of species divergence existing
% methods, which assume mutation rates are equal across taxa.
Also, by using a fully phylogenetic framework, the model can infer more
complicated patterns of divergence times than is currently possible with
existing methods restricted to comparing pairs of taxa.

\subsection*{Simultaneous Estimation of Population History and Structure: A
Novel Approach}
Currently, the inference of species trees from multi-locus genetic data
collected at a phylogeographic scale is typically conducted in a two-step
process.
First, the investigator must estimate how many populations (or species) are
represented in their sample, and assign individuals to those populations.
This is typically done using some variant of a genetic clustering algorithm.
Second, the relationships among the resulting clusters is then inferred under a
multi-species coalescent model.
The reason this two-step process is currently necessary is because current
implementations of multi-species coalescent models assume the population
assignment is known without error, and is fixed during the analysis.
This assumption is often unrealistic for empirical data, because it is often
difficult to diagnose cryptic, independent evolutionary lineages with complete
certainty.
Another potential problem with this approach is that the genetic clustering
methods simply group individuals based on allele frequencies, and do not use
the information contained in the sequence data about the genealogical history
of the alleles; using such information is a major strength of coalescent-based
methods.
Thus, initially inferring the population assignments using
allele-frequency-based methods does not make full use of the sequence data and
is more likely to lead to erroneous results.

A more robust approach would be to integrate these two steps and make
population assignment a random variable estimated during the inference of the
population tree.
I will collaborate with Mark Holder to develop a method to do this using a
nonparametric Bayesian MCMC method that treats population assignment as a
random variable under a discrete prior distribution.
There are several probability distributions (e.g. Dirichlet process, Pitman-Yor
process, and uniform process) commonly used as priors in Bayesian nonparametric
statistics in situations such as this, where we need to estimate the assignment
of random variables (gene copies) to groups (populations).
We will implement this method by incorporating a Gibbs-sampling proposal
mechanism into the BEAST MCMC machinery that numerically integrates over the
possible numbers of populations and individual assignments.
%This sampling procedure entails picking a gene copy, removing it from its
%current population and re-assigning it to one of the other populations or a
%new population according to probabilities calculated using the likelihood
%function of the *BEAST model and the current states of other parameters.
%This procedure will be repeated, sweeping across all gene copies, resulting in
%a newly proposed population structure.
%As with all other proposals, acceptance as the next state in the Markov chain
%will be governed by the Metropolis-Hastings algorithm.
This software tool will benefit a broad community of researchers conducting
phylogeographic analyses, and we plan to propose the project to the NSF
Division of Environmental Biology (DEB) Phylogenetic Systematics Program to
secure funds for the necessary computational resources.

\subsection*{Diversification and Adaptation of Lizards on Continental Islands
of the Sunda Shelf}
The Sunda Shelf of Southeast Asia has been an incredibly dynamic landscape over
the past two million years.
The extent of terrestrial habitat has fluctuated dramatically throughout the
Quaternary glacial cycles (Figure \ref{map}).
Currently surrounding mainland Southeast Asia and the major islands of the
Sunda Shelf, are a network of small continental islands that were formally part
of the Sunda Peninsula during glacial periods.
Many of these islands are merely wind-swept rocky outcrops that, nonetheless,
support small populations of skinks (\emph{Eutropis multifasciata}) and geckos
(\emph{Lepidodactylus lugubris}).
These islands were likely completely inundated during the last two
interglacial highstands ($\sim$400,000 and $\sim$120,000 years ago) when sea
levels were up to 20 meters above present.
For the next 100,000 years, these islands were part of the exposed Sunda Shelf
landmass during the Wisconsin glaciation, before being fragmented by rising sea
levels $\sim$12,000 years ago.

The lizard species inhabiting these islands have interesting biologies.
\emph{E.\ multifasciata} has occupied a distinct niche from their mainland,
forest-dwelling counterparts; they subsist off arthropods attracted to the
debris of nesting colonies of terns.
They also exhibit extreme polymorphisms in color patterns among island
populations, far exceeding the variation seen on the mainland.
Furthermore, across its broad range, \emph{E.\ multifasciata} populations vary
between egg-laying (oviparity) and live-bearing (ovoviviparity) modes of
reproduction.
%\emph{L.\ lugubris} is often restricted to peripheral habitats, such as
%wind-swept rocky islands, because it is outcompeted by other gecko species in
%more typical gecko habitats.
Similarly, populations of \emph{L.\ lugubris} also vary in reproductive mode
across the range of the species; some population are comprised solely of
asexual, parthenogenetic females, whereas others are sexual.

The network of minuscule islands distributed across the Sunda Shelf harboring
small insular populations of co-distributed lizards offers a unique model
system for studying the 
1) affects of recent, severe fragmentation on historical demography and
diversification,
2) evolution of reproductive mode,
3) rapid evolution of color pattern, and 
4) the interplay between selection and genetic drift in extremely small
populations under strong selective pressures.
My collaborators and I will foster this natural model system as part of a
long-term project to address these questions.
My collaborators include
Dr. Lee Grismer, Professor of Biology at La Sierra University, and
international colleagues
Dr. Norhayati Ahmad, Professor of Science and Technology at the Universiti
Kebangsaan Malaysia (UKM), and
Dr. Shahrul Anuar Mohd Sah, Professor of the Biological Sciences at the
Universiti Sains Malaysia (USM).
We plan to approach different aspects of this study in a series of stages.

The first stage will entail next-generation sequencing of hundreds of loci from
dense sampling of individuals of both species across the islands and
surrounding landmasses, and collecting a database of whole genomes and
transcriptomes from a subset of individuals from a few of the well-sampled
islands of known reproductive mode and color phenotype.
With the multi-locus data, we will elucidate the evolutionary context of the
island populations by using coalescent-based models to infer their past
demography and relationships.
Not only will this address interesting questions about the evolution of
populations in this dynamic fragmented landscape, but it will also create an
evolutionary framework to guide additional data collection and analyses.
It will allow us to appropriately model the genetic covariance among
populations in all downstream analyses, avoiding the erroneous assumption of
independence among populations.
Using the preliminary genomic and transcriptomic data, we will begin to
characterize the genetic underpinnings of the variable natural-history
characteristics of both species (reproductive modes and color patterns).
We will determine whether the phenotypic variation for each trait is
environmentally or genetically based, and if the latter, whether it is
predominantly the result of variation in coding or regulatory sequences.
We will seek funding from the NSF Phylogenetic Systematics Program for this
initial stage of the project.

Once we identify the regions of the genome relevant to the variation in the
traits of interest, we will begin the second stage of the project, in which we
will collect sequence data from these regions from dense population-level
sampling across the islands and neighboring mainland.
This stage will also entail field and lab observations to characterize the
traits of interest in each population.
Using these data, we can determine if the genetic variants responsible for the
different phenotypes are the result of independent, parallel mutations, or
whether these variants are present across much of the distribution of these
species at low frequencies and are independently fixed among different
populations.
We will also analyze these data within the phylogeographic context of the
populations elucidated in Stage 1 to understand the relative contributions of
selection versus drift in the evolution of these traits.
These populations have recently been restricted to the unusual, harsh habitats
of the Sunda Shelf islands and have likely been experiencing strong selective
pressures as a result.
At the same time, genetic drift is undoubtedly an extremely strong evolutionary
force on these islands where populations consist of only hundreds of
breeding adults.
This offers a natural model system for studying the interplay between the
deterministic force of selection and the stochastic force of drift in shaping
the genomic landscapes and phenotypic characteristics of these populations.
We will seek funding from the NSF Phylogenetic Evolutionary Genetics Program
for the second stage of the project.

Once we have established the phylogeographic context of the island populations
and the genetic mechanisms responsible for their phenotypic variation, the
third stage of the project will aim to understand the evolutionary ecology of
this system.
We want to understand what intra-specific, inter-specific, and abiotic
interactions influence the selection on the regions of the genome that control
important phenotypic characteristics.
For example, are there particular ecological contexts that favor
parthenogenesis in \emph{L.\ lugubris}?
Are there particular inter-specific interactions that favor ovoviviparity in
\emph{E.\ multifasciata}, like introduced mammalian predators of lizard eggs?
Are the striking color polymorphisms of \emph{E.\ multifasciata} adaptive?
From the fieldwork and sequence collection associated with the first to stages
of this project, we hope to sufficiently characterize the ecosystems of these
islands and have the necessary genomic data to allow us to assess such
questions at the genetic level in these species.
We feel this third stage will be a strong candidate for the NSF Evolutionary
Ecology Program.

\end{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
